File: Inicio.py
#Entrada de datos: 
Nombre = str(input("Ingresa tu nombre: "))
Edad = int(input("Ingresa tu edad: "))
Salario = float(input("Ingresa tu salario mensual: "))
HorasTrabajadas = int(input("Ingresa tus horas trabajadas por semana: "))
#Funciones: 
Edad5Años = Edad+5
SalarioAnual = Salario*12
SalarioHora = (Salario/31)/12
#Salida:
print(" ¡¡HOLA "+Nombre+"!! ")
print(" ¡¡En 5 años tendrás "+str(Edad5Años)+" años!! ")
print(" ¡¡Tu salario anual será de: "+str(SalarioAnual)+" pesos!! ")
print(" ¡¡Tu salario por hora es de: "+str(SalarioHora)+" pesos!!")

File: Inicio.cpp
#include <iostream>

using namespace std;
int main(){
    string Nombre;
    int NumeroControl;
    string Carrera;
    cout<<"Ingresa tu nombre: "<<endl;
    cin >> Nombre;
    cout<<"Ingresa tu número de control: "<<endl;
    cin >> NumeroControl;
    cout<<"Ingresa tu carrera: "<<endl;
    cin >> Carrera;
    cout << "¡¡Hola "<<Nombre<<"!!"<<endl;
    cout << "Tu número de control es "<<NumeroControl<<endl;
    cout << "Tu carrera es "<<Carrera<<endl;
    return 0;
}